quarkus-opentelemetry-exporter-gcp/deployment/src/test/java/io/quarkiverse/opentelemetry/exporter/gcp/deployment/GcpExporterEnabledTest.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkiverse.opentelemetry.exporter.gcp.deployment;
+
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.quarkiverse.opentelemetry.exporter.gcp.runtime.LateBoundBatchSpanProcessor;
+import io.quarkiverse.opentelemetry.exporter.gcp.runtime.LateBoundSimpleSpanProcessor;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class GcpExporterEnabledTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest config = new QuarkusUnitTest()
+            .withEmptyApplication()
+            .overrideConfigKey("quarkus.opentelemetry.tracer.exporter.gcp.enabled", "true")
+            .overrideConfigKey("quarkus.opentelemetry.tracer.exporter.gcp.projectid", "test");
+
+    @Inject
+    OpenTelemetry openTelemetry;
+
+    @Inject
+    Instance<LateBoundBatchSpanProcessor> lateBoundBatchSpanProcessorInstance;
+
+    @Inject
+    Instance<LateBoundSimpleSpanProcessor> lateBoundSimpleSpanProcessors;
+
+    @Test
+    void testOpenTelemetryButNoBatchSpanProcessor() {
```

Comment:
If credentials are mandatory, the test must not be active by default or the CI will fail... It should be available for manual testing, though. 
I wonder if a different test can mock the google cloud part, in order to allow an always active test.